include/nbl/ext/FullScreenTriangle/FullScreenTriangle.h

```diff
@@ -96,11 +102,16 @@ namespace nbl
 				records.
 			*/
 
-			inline bool recordDrawCalls(video::IGPUCommandBuffer* commandBuffer)
+			inline bool recordDrawCalls(core::smart_refctd_ptr<nbl::video::IGPUGraphicsPipeline> gpuGraphicsPipeline, core::smart_refctd_ptr<nbl::video::ISwapchain> swapchain, video::IGPUCommandBuffer* commandBuffer)
 			{
 				_NBL_STATIC_INLINE_CONSTEXPR auto VERTEX_COUNT = 3;
 				_NBL_STATIC_INLINE_CONSTEXPR auto INSTANCE_COUNT = 1;
 
+				auto layout = gpuGraphicsPipeline->getRenderpassIndependentPipeline()->getLayout();
+				const asset::SPushConstantRange& pcRange = layout->getPushConstantRanges().begin()[0];
+				uint32_t surfaceTransform = swapchain->getSurfaceTransform();
```

Comment:
we use the function for post processing into a rendertarget, don't ask for swapchain, just as for a display orientation